From ae9ac0aaba712fa5d13070ecda583815228db308 Mon Sep 17 00:00:00 2001
From: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Date: Mon, 5 Sep 2022 11:34:22 +0300
Subject: [PATCH 096/102] arm: s32-cc: Increase max DTB size to 0x8000

This is needed to keep U-Boot and ATF in sync.

Issue: ALB-9269
Upstream-Status: Pending 

Signed-off-by: Ghennadi Procopciuc <ghennadi.procopciuc@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 arch/arm/mach-s32/s32-cc/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-s32/s32-cc/Kconfig b/arch/arm/mach-s32/s32-cc/Kconfig
index a9c68cbe99..ff628f57aa 100644
--- a/arch/arm/mach-s32/s32-cc/Kconfig
+++ b/arch/arm/mach-s32/s32-cc/Kconfig
@@ -39,7 +39,7 @@ config SYS_DATA_BASE
 
 config S32CC_MAX_DTB_SIZE
 	hex "Max size reserved for device tree in image layout"
-	default 0x7000
+	default 0x8000
 
 config SYS_MALLOC_F_LEN
 	default 0x2000
-- 
2.17.1

